packages/lab/stories/multiline-input/multiline-input.stories.tsx

```diff
@@ -100,6 +100,38 @@ export const Disabled: ComponentStory<typeof MultilineInput> = (args) => {
     </FlowLayout>
   );
 };
+
+export const CharacterCount: ComponentStory<typeof MultilineInput> = (args) => {
+  const [value, setValue] = useState<string>("Value");
+  const MAX_CHARS = 10;
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const newVal = event.target.value;
+
+    if (newVal.length <= MAX_CHARS) {
+      const value = newVal;
+      setValue(value);
+    }
+  };
+
+  return (
+    <MultilineInput
+      {...args}
+      value={value}
+      onChange={handleChange}
+      endAdornment={
+        <span
```

Comment:
More of a minor thing but would using Text have been better here? They wouldn't need to know so much about the CSS variables

Reply:
changed :) good shout